docs/book/supported_features_matrix.md

```diff
@@ -2,19 +2,28 @@
 
-Note: Project pacific and Tanzu Kubernetes Grid support only block volumes.
+_Notes_:
 
-| Feature                     | Vanilla Block                                                            | Vanilla File   | Project pacific Block | Tanzu K8s Grid for Project pacific Block |
-|-----------------------------|--------------------------------------------------------------------------|----------------|-----------------------|------------------------------------------|
-| Dynamic Provisioning        | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
-| vSphere UI integration      | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
-| Multi-master for CSI driver | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
-| Datastores supported        | vmfs/nfs/vSAN/(vVOL `*`)                                                 | vSAN `*`       | vmfs/nfs/vSAN `*`     | vmfs/nfs/vSAN `*`                        |
-| Static Provisioning         | Yes                                                                      | Yes `*`        | No                    | Yes                                      |
-| Access mode                 | RWO                                                                      | ROX,RWX `*`    | RWO*                  | RWO `*`                                  |
-| Volume Topology/Zones       | Yes (beta)                                                               | No             | No                    | No                                       |
-| Extend Volume               | Yes (k8s 1.16+)* <br>beta                                                   | No             | No                    | No                                       |
-| Encryption                  | Yes (SPBM policy driven)*                                                | No             | No                    | No                                       |
-| K8s versions supported      | 1.14, 1.15 for CSI 1.0 features<br>1.15, 1.16, 1.17 for CSI 2.0 features | 1.16.x, 1.17.x | 1.16.x                | 1.16.x                                   |
+* If the CSI version `2.0` driver is installed on K8s running on vSphere 6.7U3, the older CSI `1.0` driver features continue to work but the new CSI `2.0` features are not supported.
+* If the CSI version `1.0.2` is installed on K8s running on vSphere 7.0, the CSI `1.0` driver features continue to work. CSI version `1.0.1` is not compatible with vSphere 7.
+* CSI version `1.0.x` and CSI version `2.0` on vSphere 6.7U3, vSphere with K8s, TKG ‘Guest’ and PKS 1.7 only support dynamically provisioned block (`RWO`) volumes on vSphere storage.
```

Comment:
CSI 1.0.x and 2.0.0 releases does not match with vSphere with k8s and TKG guest releases. Basically CSI 1.0.x and 2.0.0 releases available in GitHub is only for vanilla k8s.